test/extensions/resource_monitors/fixed_heap/fixed_heap_monitor_test.cc

```diff
@@ -0,0 +1,28 @@
+#include "extensions/resource_monitors/fixed_heap/fixed_heap_monitor.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace ResourceMonitors {
+namespace FixedHeapMonitor {
+
+TEST(FixedHeapMonitorTest, SaneUsage) {
+  envoy::config::resource_monitor::fixed_heap::v2alpha::FixedHeapConfig config;
+  config.set_max_heap_size_bytes(std::numeric_limits<uint64_t>::max());
+  std::unique_ptr<FixedHeapMonitor> monitor(new FixedHeapMonitor(config));
```

Comment:
Could `FixedHeapMonitor` receive its memory query functions as a mockable object in its constructor? This would allow you to validate the actual math in the utilization function as well with mockable inputs.